cipher/rsa/rsa_test.go

```diff
@@ -35,30 +35,27 @@ var rsaTestData = []struct {
 	},
 }
 
-func TestEncryptDecrypt(t *testing.T) {
+func testPrecondition(t *testing.T) (int64, int64, int64) {
 	// Both prime numbers
 	p := int64(61)
 	q := int64(53)
-
 	n := p * q
-
 	delta := lcm.Lcm(p-1, q-1)
-
 	e := int64(17) // Coprime with delta
-
 	if gcd.Recursive(e, delta) != 1 {
 		t.Fatal("Algorithm failed in preamble stage:\n\tPrime numbers are chosen statically and it shouldn't fail at this stage")
```

Comment:
```suggestion
		t.Fatal("something went wrong: prime numbers are chosen statically and it shouldn't fail at this stage")
```

Reply:
Not that big a difference but a meaningful one - the Go test output shouldn't have newlines and/or tabs (not something you did but if you can change it 🙏🏼)